docs/DeveloperGuide.md

```diff
@@ -258,12 +258,13 @@ _{Explain here how the data archiving feature will be implemented}_
 **Target user profile**:
 
 * has a need to manage a significant number of contacts
+* has a need to aggregate links and social media handles of contacts
 * prefer desktop apps over other types
 * can type fast
 * prefers typing to mouse interactions
 * is reasonably comfortable using CLI apps
 
-**Value proposition**: manage contacts faster than a typical mouse/GUI driven app
+**Value proposition**: manage links and contacts faster than a typical mouse/GUI driven app
 
```

Comment:
My suggestion: organise contacts and meetings across various social media platforms ***so that they can be retrieved more conveniently when needed***